setup.py

```diff
@@ -29,7 +29,9 @@
     description='A robot-agnostic teleoperation node to convert keyboard'
                 'commands to Twist messages.',
     license='BSD',
-    tests_require=['pytest'],
+    extras_require={
+        'test': ['pytest'],
+    }, 
```

Comment:
```suggestion
    },
```

That should fix the CI failure.